### DISCUSSION TITLE
How should I modify the model ?

### DISCUSSION THREAD

**Discussion by jzdcf** [openvinotoolkit/nncf#2799] [Q&A]
version 2024
quantized_model = nncf.quantize(model, calibration_dataset)

Quantifying in pytorch is successful, The graph seems to meet the quantification criteria
The ONNX model was used for inference and the expected output was also obtained
When running the above code, different models report errors as follows：

model_pytorch:
Only Tensors created explicitly by the user (graph leaves) support the deepcopy protocol at the moment.  If you were attempting to deepcopy a module, this may be because of a torch.nn.utils.weight_norm usage, see https://github.com/pytorch/pytorch/pull/103001

model_onnx:
nncf.errors.ValidationError: There is no tensor with the name None

**Comment by alexsu52** [comment]
Hello @jzdcf! Could you share the code to reproduce the error?
